package.json

```diff
@@ -75,12 +75,12 @@
     "semver": "^5.3.0",
     "shelljs": "^0.8.3",
     "url-loader": "^0.5.7",
-    "vue": "^2.5.15",
+    "vue": "^2.5.22",
```

Comment:
Could we also bump this to latests Vue 2.x?

Reply:
This is all updated, and works fine